server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreClusterStateRestoreIT.java

```diff
@@ -134,88 +102,10 @@ public void testFullClusterRestoreMultipleIndices() throws Exception {
         assert !Objects.equals(newClusterUUID, prevClusterUUID) : "cluster restart not successful. cluster uuid is same";
 
         // Step - 3 Trigger full cluster restore
-        restoreAndValidate(prevClusterUUID, indexStats);
-        ensureGreen(secondIndexName);
-        verifyRestoredData(indexStats2, secondIndexName);
-    }
-
-    @AwaitsFix(bugUrl = "https://github.com/opensearch-project/OpenSearch/issues/9834")
-    public void testFullClusterRestoreFailureValidationFailures() throws Exception {
```

Comment:
why did we remove the failure cases here?

Reply:
this should not be removed. let me check and revert